portfolio/frontend/src/containers/Login/Login.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import classes from "./Login.module.css"
+
+const Login = (props) => {
+    return (
+        <div>
+            <div className={classes.Header}>
+                <img src="sGonksLogo.png" alt="sGonks Logo" className={classes.Logo}></img>
+                <div className={classes.HeaderButtons}>
+                    <button className={classes.Login} onClick={() => props.handler()}>Login</button>
+                    <button className={classes.Help}>See how it works</button>
+                </div>
+            </div>
+
+            <div className={classes.Body}>
+                <div className={classes.Column}>
+                    <h1>Who's the next trendsetter?</h1>
+                    <p>Bacon ipsum dolor amet ground round chislic bresaola, andouille pig buffalo swine t-bone strip steak. 
+                        Kevin shoulder frankfurter, venison short loin t-bone shank chislic chuck beef ribs pork buffalo. 
+                        Rump hamburger jowl cow. Tail ribeye burgdoggen tongue. Chislic pork belly doner rump frankfurter. 
+                        Jowl meatloaf ribeye beef pork, shoulder bresaola filet mignon.</p>
+                    <div className={classes.Buttons}>
+                        <button className={classes.Login}>Login</button>
```

Comment:
Might wanna add the `onClick={() => props.handler()` here as well so this button redirects tooo

Reply:
Oh yeah...

Reply:
Ok, fixed that. I'm failing the javascript styling check on the code you gave me, but I'm guessing that's for the reasons you mentioned about it being bad and temporary?

Reply:
The styling checks are purely cosmetic - `make pretty`should make them go away now that `clang-format` is working.

Reply:
make pretty introduces a number of errors for me currently, it's doing some weird things like replacing some > with '.

Reply:
I suppose i can just keep the changes it makes on this file for now.

Reply:
Ok I've done that and pushed the style changes.